graph/src/blockchain/mod.rs

```diff
@@ -149,6 +149,12 @@ pub trait Blockchain: Debug + Sized + Send + Sync + Unpin + 'static {
 
     type NodeCapabilities: NodeCapabilities<Self> + std::fmt::Display;
 
+    fn is_capabilities_fulfilled(
```

Comment:
I think the right place for this is on the NodeCapabilities trait since the Blockchain already contains a NodeCapabilities associated type

Reply:
Seems like a good idea. 

While checking to do it, I hit a small road block though. My current implementation on Ethereum relies on the `self.firehose_endpoints` which is hold by `chain`.

If I move it to `NodeCapabilities` I'm not sure how I would access this. In the chain agnostic `instance_manager.rs`, the call site looks like:

```
        if !chain.is_capabilities_fulfilled(&required_capabilities, &logger) {
            return Err(anyhow!(
                "deployment {} with required capabilities {} cannot be fulfilled by this instance",
                &deployment,
                &required_capabilities
            ));
        }
```

Do you think we have ways around to fix this?